src/main/java/io/jenkins/plugins/ml/IPythonKernelInterpreter.java

```diff
@@ -25,9 +25,9 @@
 package io.jenkins.plugins.ml;
 
 import org.apache.zeppelin.interpreter.*;
-import org.apache.zeppelin.jupyter.JupyterInterpreter;
 import org.apache.zeppelin.resource.LocalResourcePool;
 import org.apache.zeppelin.resource.ResourcePool;
+import org.apache.zeppelin2.jupyter.JupyterInterpreter;
```

Comment:
Here I am using the 2nd version. So it works as we expected.

Reply:
If that's the case, we might as well move that to our packages, under io.jenkins.plugins.ml. WDYT?

Reply:
Okay @kinow we can move into out packages and set the licence to AL. :+1: